pages/admin/index.tsx

```diff
@@ -57,15 +60,16 @@ function renderName({ value }: NameProps) {
   return (
     <div>
       <Text>{value.name}</Text>
-      <Text textStyle="caption" textColor="secondary">
-        ID: {value.id}
-      </Text>
+      {value.id && (
```

Comment:
hm shouldn't there always be an ID for an application?

Reply:
The ID is the applicant ID, not the application ID, so if someone is applying for the first time, they won't have an applicant ID yet.